### DISCUSSION TITLE
Example presets broken

### DISCUSSION THREAD

**Discussion by JoFaHD** [fastfetch-cli/fastfetch#1099] [Q&A]
I tried the example configurations from the presets with `fastfetch --config x.jsonc` and a few of them did not work. They threw the error: `JsonConfig Error: Unknown display property`
I think this error is due to the "constants" display property that was set and used in these examples. When I removed it the preset worked, as in it did not throw an error and showed the output. It was broken though, as I had removed the constants. I wanted to look up that property in the documentation but it wasn't even there.
Now I don't know if this is something from my part or a bug with the program or preset.
This happened in the the preset-examples 2, 10, 12, 17, 18, 20, 21, 22

From the preset 2.jsonc:
```
 "display": {
        "separator": "  ",
        "constants": [
            "───────────────"
        ]
    },
```
The constant is then used with `{$1}` think: 
```
    "modules": [
        {
            "type": "custom", // HardwareStart
            // {#1} is equivalent to `\u001b[1m`. {#} is equivalent to `\u001b[m`
            "format": "┌{$1} {#1}Hardware Information{#} {$1}┐"
        },
```

**Comment by CarterLi** [comment]
You should not download the presets from dev branch. They are in development and may use unreleased features.

All examples have been installed into your local machine when you install fastfetch. Please try `fastfetch -c examples/N` instead.

**Reply by JoFaHD** [reply]
Ah, I see. Thank you very much!
Where can I find the config-files for these?

**Reply by CarterLi** [reply]
It depends on the distro you use. Usually be in `/usr/share/fastfetch/presets/examples/`

**Reply by JoFaHD** [reply]
Perfect, thank you!
